### PR TITLE
doc: move releasers to emeritus

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://github.com/nodejs/Release/issues/1036

### CODE REVIEW

**Review by UlisesGascon — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -219,7 +216,9 @@ and CITGM team members listed below.
 
 ### LTS team
 - [@addaleax](https://github.com/addaleax) - Anna Henningsen
+* [@BethGriggs](https://github.com/BethGriggs) - Bethany Griggs
```
Comment (richardlau) [inline README.md:219]: Should be (also?) in the emeritus releasers list.

File: README.md [hunk 2]
```diff
@@ -219,7 +216,9 @@ and CITGM team members listed below.
...
 - [@bnoordhuis](https://github.com/bnoordhuis) - Ben Noordhuis
+* [@danielleadams](https://github.com/danielleadams) - Danielle Adams
```
Comment (richardlau) [inline README.md:221]: Should be (also?) in the emeritus releasers list.

**Comment by RafaelGSS** [comment]
> Should be (also?) in the emeritus releasers list.

I'm not sure, there are emeritus in one list but not in others

**Comment by richardlau** [comment]
> > Should be (also?) in the emeritus releasers list.
> 
> I'm not sure, there are emeritus in one list but not in others

As former releasers, they should definitely be in releasers emeritus.

Beth and Danielle were also in the LTS team, prior to us removing the team in https://github.com/nodejs/Release/pull/1037, so I think it would also make sense for them to be in LTS emeritus as well: https://github.com/nodejs/Release/blob/143ae025dacdc4d1b08ba32365e083f76a0ab053/README.md?plain=1#L183-L196

**Review by richardlau — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Pull request merged**